pkg/capture/utils/filename.go

```diff
@@ -0,0 +1,27 @@
+package utils
```

Comment:
Having a `utils` package at all is kind of an antipattern. I think what you have here can be done with a type:

```go
package capture

// snip

type Time struct {
       time.Time
}

func (t Time) String() {
     // inline all the functions you have below in here
}

func NewTime(t string) (Time, error) {
     // everything from ConvertStringToTimestamp
}
```

Then we have our idea of what a `Time` is in the context of the capture package. This also makes it more type-oriented than procedural.

Reply:
Thanks, changed the code to take this approach. Let me know what you think :)